fab/fabfile.py

```diff
@@ -728,7 +729,7 @@ def get_number_of_releases():
 def mark_last_release_unsuccessful():
     # Removes last line from RELEASE_RECORD file
     with cd(env.root):
-        sudo("sed -i .bak '$d' {}".format(RELEASE_RECORD))
+        sudo("sed -i '$d' {}".format(RELEASE_RECORD))
```

Comment:
for some reason ubuntu chokes on the '.bak' where mac requires it (where i do my testing)